File: BankKycSystemdotnet/BankKycSystem/DTOs/KycApplicationDTO.cs
namespace BankKycSystem.DTOs
{
    public class KycApplicationDTO
    {
        public long Id { get; set; }
        public string? CustomerName { get; set; }
        public string? PoiDocumentPath { get; set; }
        public string? PoaDocumentPath { get; set; }
        public string? Remarks { get; set; }
        public string? Status { get; set; }
        public DateTime? SubmittedAt { get; set; }
        public string? AccountType { get; set; }
    }
}

File: bank-kyc-system/src/main/java/com/bank/project/dto/KycApplicationDTO.java
// com.bank.project.dto.KycApplicationDTO.java

package com.bank.project.dto;

import com.bank.project.enums.AccountType;
import com.bank.project.enums.KycStatus;
import com.fasterxml.jackson.annotation.JsonIgnore;

import lombok.*;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class KycApplicationDTO {
    private Long id;
    private String poiDocumentPath;
    private String poaDocumentPath;
    private KycStatus status;
    private String remarks;
    private LocalDateTime submittedAt;
    private AccountType accountType;

    @JsonIgnore   
    private Long userId;

    private String userName;
}
